config/cryptonodes/parity/parity.toml

```diff
@@ -0,0 +1,37 @@
+[parity]
+mode = "active"
+mode_timeout = 300
+mode_alarm = 3600
+auto_update = "critical"
+release_track = "stable"
+public_node = false
+no_download = false
+no_consensus = false
+base_path = "/parity/.local/share/io.parity.ethereum"
```

Comment:
This doesn't match the actual base parity directory you're using in compose which is `/home/parity`
I'd suggest simply removing these lines